packages/amplify-cli/src/input-manager.ts

```diff
@@ -206,3 +211,47 @@ function aliasArgs(argv: string[]) {
     insertAmplifyIgnore(gitIgnoreFilePath);
   }
 }
+
+const convertKeysToLowerCase = <T>(obj: Record<string, T>): Record<string, T> => {
+  const newObj = {};
+  Object.entries(obj).forEach(([key, value]) => { newObj[key.toLowerCase()] = value; });
+  return newObj;
+};
+
+const normalizeStatusCommandOptions = (input: Input): Input => {
+  const options = input.options ? input.options : {};
+  const allowedVerboseIndicators = [constants.VERBOSE, 'v'];
+  // Normalize 'amplify status -v' to verbose, since -v is interpreted as 'version'
+  allowedVerboseIndicators.forEach(verboseFlag => {
+    if (options[verboseFlag] !== undefined) {
```

Comment:
main change was changing from `options.verboseFlag` to `options[verboseFlag]`, everything else is just moving the method to another file to keep input logic in a single file